### PR TITLE
Refactor RedisService

### PR DESCRIPTION
**Pull request by hishamco**

### CODE REVIEW

File: src/OrchardCore.Modules/OrchardCore.Redis/Services/RedisService.cs [hunk 1]
```diff
@@ -10,21 +10,23 @@ namespace OrchardCore.Redis.Services
...
+            _options = options.Value;
             _logger = logger;
 
             InstancePrefix = options.Value.InstancePrefix;
         }
 
         public IConnectionMultiplexer Connection { get; private set; }
-        public string InstancePrefix { get; private set; }
+
+        public string InstancePrefix { init; get; }
```
Comment (MikeAlhayek) [inline src/OrchardCore.Modules/OrchardCore.Redis/Services/RedisService.cs:28]: Why do you need this change?

Comment (hishamco) [reply src/OrchardCore.Modules/OrchardCore.Redis/Services/RedisService.cs:28]: `InstancePrefix` is only readonly and set in the constructor, right?

Comment (MikeAlhayek) [reply src/OrchardCore.Modules/OrchardCore.Redis/Services/RedisService.cs:28]: yes

Comment (jtkech) [reply src/OrchardCore.Modules/OrchardCore.Redis/Services/RedisService.cs:28]: So you can remove the init only setter as we will never use an object initializer on `RedisService`, `new RedisService(...) { InstancePrefix = ... }`, so I suggest.

    public string InstancePrefix { get; }

**Review by MikeAlhayek — APPROVED** [review]

**Pull request merged**